docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -32,24 +32,26 @@ Prerequisites
 Software
 ~~~~~~~~
 
-- A :doc:`Flocker development machine <vagrant>`.
 - A web browser.
-- An up-to-date clone of the `Flocker repository <https://github.com/ClusterHQ/flocker.git>`_.
-- An up-to-date clone of the `homebrew-tap repository <https://github.com/ClusterHQ/homebrew-tap.git>`_.
 - `gsutil Python package <https://pypi.python.org/pypi/gsutil>`_ on your workstation.
+- `Vagrant`_ (1.6.2 or newer)
+- `VirtualBox`_
+
+.. _`Vagrant`: https://docs.vagrantup.com/
+.. _`VirtualBox`: https://www.virtualbox.org/
 
 Access
 ~~~~~~
 
 
-- Access to `Google Cloud Storage`_ using `gsutil`_ on your workstation and your :doc:`Flocker development machine <vagrant>`.
+- Access to `Google Cloud Storage`_ using `gsutil`_ on your workstation.
```

Comment:
It used to be that before starting you:
* Create a dev box
* Set up gsutil on the dev box
* Set up gsutil on your workstation

Now before starting you:
* Set up gsutil on your workstation

then during the process you:
* Create a dev box
* Set up gsutil on the dev box

Because creating a dev box is no longer a prerequisite and is instead part of the process, setting up gsutil on that dev box is now part of the process and not a prerequisite.